create_derivatives/routines.py

```diff
@@ -156,4 +157,12 @@ def run(self, replace):
 
 
 class CleanupRoutine:
```

Comment:
Does "cleanup" by definition mean to remove stuff? I was going to suggest being explicit in a docstring and/or the `msg` about what cleanup meant, but maybe that doesn't need defining?

Reply:
You're right that a docstring would be helpful. In the context of seeing this message in the wild (e.g., in Zodiac) I think it's sufficient, but I also don't have particularly strong feelings about what the message says.